crates/router/src/services/authorization/info.rs

```diff
@@ -16,6 +18,25 @@ pub fn get_group_authorization_info() -> Vec<GroupInfo> {
         .collect()
 }
 
+pub fn get_group_authorization_info_with_group_tag() -> Vec<ParentInfo> {
+    PermissionGroup::iter()
+        .map(|value| (get_parent_name(value), value))
+        .fold(
+            HashMap::new(),
+            |mut acc: HashMap<ParentGroup, Vec<PermissionGroup>>, (key, value)| {
+                acc.entry(key).or_default().push(value);
+                acc
+            },
+        )
+        .into_iter()
+        .map(|(name, value)| ParentInfo {
+            name: name.clone(),
+            description: get_parent_group_description(name),
+            groups: value,
+        })
+        .collect()
+}
+
```

Comment:
Can we move this logic to core function itself. This is neither a util nor being used in this file.

This will also remove the usage of map in the core function.